### ISSUE TITLE
Setting a passphrase and virtual box driver twice causes the socket to be unreachable 

### ISSUE THREAD

**Issue by rlewkowicz** [canonical/multipass#2589]
```
multipass set local.passphrase=phrase
multipass authenticate phrase
multipass set local.driver=virtualbox

multipass set local.passphrase=phrase
multipass authenticate phrase
multipass set local.driver=virtualbox

multipass ls
```

for now, I'l just check if it's set, but it should be idempotent

**Comment by rlewkowicz** [comment]
There's a lot of wonkey stuff going on with multipass. It's a good product, but there's still rough edges. It was really nice to be able to switch to vbox though almost transparently. I don't see a response, and my replication here might not be enough to warrant one. I'm just closing this to reduce noise on the project.

**Comment by townsend2010** [comment]
Hey @rlewkowicz,

We are a small team and have had other things going on before we can triage this bug (and sometimes things just get missed until a later time). That said, I think this issue is worth looking in to.  The daemon should not become unresponsive with the commands you posted.  I'm reopening this in order for us to look at it sometime.

**Comment by tobe2098** [comment]
This is still a relevant issue, although it is low priority due to the planned deprecation of VirtualBox.

**Comment by xmkg** [comment]
This is unlikely to be fixed since VirtualBox driver is to be removed in a future release.
